bindings/wasm/benchmark_test.go

```diff
@@ -26,10 +25,7 @@ func BenchmarkExample(b *testing.B) {
 		b.Fatal(err)
 	}
 
-	request := &bindings.InvokeRequest{
-		Metadata:  map[string]string{"args": "salaboy"},
```

Comment:
No more @salaboy? :)

Reply:
:) I decided to use the benchmark to test baseline overhead, like the minimal for any wasi function. salaboy still lives forever in the test cases :D

Reply:
😂